### PR TITLE
Move domainId from Meta to Meta.Source

### PR DESCRIPTION
**Pull request by Andrey9kin**
DomainId seems to be belong to the source structure since it points out from which domain this message was sent to

### CODE REVIEW

**Comment by d-stahl-ericsson** [comment]
Makes sense to me.

File: eiffel-syntax-and-usage/the-meta-object.md [hunk 1]
```diff
@@ -44,6 +38,12 @@ __Format:__
 __Required:__ No  
 __Description:__ A description of the event sender. Primarily for traceability purposes.
 
+#### meta.domainId
```
Comment (p-backman-ericsson) [inline eiffel-syntax-and-usage/the-meta-object.md]: Change meta.domainId => meta.source.domainId

**Comment by d-stahl-ericsson** [comment]
Proposal: 
We move domainId into meta.source. We make meta.source required, and domainId required. Other source children are optional. When we do that, let's also improve on the documentation of domainId, strongly encouraging coherent formatting, e.g. "Java package notation", e.g. com.ericsson.whatever.

Can you update the PR with this @Andrey9kin? Then we'll merge it.

**Comment by Andrey9kin** [comment]
@d-stahl-ericsson please take a look 2d60943

File: eiffel-syntax-and-usage/the-meta-object.md [hunk 2]
```diff
@@ -41,9 +35,15 @@ __Description:__ Any tags or keywords associated with the events, for searchabil
...
 __Format:__  
-__Required:__ No  
+__Required:__ Yes
 __Description:__ A description of the event sender. Primarily for traceability purposes.
 
+#### meta.source.domainId
+__Type:__ String  
+__Format:__ Free text  
+__Required:__ Yes  
+__Description:__ Identifies the domain that produced an event. A domain is an infrastructure topological concept, which may or may not corresponds to an organization or product structures. A good example would be Java packages notation, ex. com.mycompany.product.component or mycompany.site.division. Also, keep in mind, that division names, as well as product/component names, tends to change. That is why it might be a good idea to pick coded names that do not directly reflect current naming in the organization. Think of flower names as names for sites or simply site1, site2, site3 instead of Sweden, Denmark, Norway.
```
Comment (d-stahl-ericsson) [inline eiffel-syntax-and-usage/the-meta-object.md]: I would argue that physical sites and products are rather good examples of entities to put into a domainId, as opposed to organization. If you go all abstract and speak in code, the usefulness is arguably not that great... :)

And then if you move your equipment from Sweden to Denmark, then you can of course always change your domainId. And perhaps you should!

**Comment by danielyinanc** [comment]
:+1:

**Comment by d-stahl-ericsson** [comment]
I have created issue #35 to track this change. I will close this PR with the follow recommendations, @Andrey9kin.

- Change the description as per comment above.
- Make a new PR to topic-drop3, including changes to examples.
- Rebase it to forthcoming change to move eiffel-examples and eiffel-schemas into this repository (#26). We'll try to move quickly with that one to not cause any delays.

**Pull request closed without merging**